backend/cmd/user/root.go

```diff
@@ -7,7 +7,7 @@ import (
 func NewUserCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "user",
-		Short: "User import/export(TODO) tools",
+		Short: "User import/export tools",
 		Long:  `Add the ability to import users into the hanko database.`,
```

Comment:
The long description should also reflect that there is an export function now.

Reply:
@FreddyDevelop done!